pennylane/io.py

```diff
@@ -77,14 +78,23 @@ def load(quantum_circuit_object, format: str, **load_kwargs):
             Currently, only supported for Qiskit's `QuantumCircuit <https://docs.pennylane.ai/projects/qiskit>`_.
 
     Returns:
-        function: the PennyLane template created from the quantum circuit
-        object
+        function: the PennyLane template created from the quantum circuit object
+
+    .. warning::
+        ``qml.load`` is deprecated and will be removed in version 0.37. Instead, please
+        use the appropriate ``qml.from_x`` functions, such as ``qml.from_qiskit``.
+
     """
 
+    warnings.warn(
+        "qml.load is deprecated and will be removed in version 0.37. Instead, "
+        "please use the appropriate qml.from_x functions, such as qml.from_qiskit.",
```

Comment:
Since we know the target `format`, we could give a specific recommendation here.